src/Loggable/Mapping/Driver/Xml.php

```diff
@@ -85,10 +85,8 @@ public function readExtendedMetadata($meta, array &$config)
 
     /**
      * Searches mappings on element for versioned fields
-     *
-     * @param object $meta
      */
-    private function inspectElementForVersioned(\SimpleXMLElement $element, array &$config, $meta)
+    private function inspectElementForVersioned(\SimpleXMLElement $element, array &$config, object $meta): void
```

Comment:
```suggestion
    private function inspectElementForVersioned(\SimpleXMLElement $element, array &$config, ClassMetadata $meta): void
```
importing `Doctrine\Persistence\Mapping\ClassMetadata`